### PR TITLE
Replace IDs in HTML elements with css class

### PR DESCRIPTION
**Pull request by marionb**
Pleas review.

### CODE REVIEW

**Comment by pgiraud** [comment]
Can you please rebase your branch?

**Comment by marionb** [comment]
Pleas review.

File: thinkhazard/templates/index.jinja2 [hunk 1]
```diff
@@ -18,15 +18,15 @@
       <h3>Identify the hazards in your project area and receive guidance on how to handle them</h3>
     </header>
   </div>
-  <div id="search-pane" class="container-fluid search-pane">
+  <div class="search-pane container-fluid search-pane">
```
Comment (pgiraud) [inline thinkhazard/templates/index.jinja2]: `search-pane` duplicated

File: thinkhazard/templates/report.jinja2 [hunk 2]
```diff
@@ -61,9 +61,9 @@ Think Hazard - {{ division.name}}
       </h2>
 
       {% if current_hazard %}
-      <a id="overview-link" class="btn btn-default btn-xs pull-left" href="{{ overview_url }}" title="Go back to overview page"><i class="fa fa-home"></i> Overview</a>
+      <a class="btn btn-default btn-xs pull-left" href="{{ overview_url }}" title="Go back to overview page"><i class="fa fa-home"></i> Overview</a>
       {% endif %}
-      <ul id="hazard-types-list" class="horizontal list-unstyled hazards" role="tablist">
+      <ul class="hazard-types-list horizontal list-unstyled hazards" role="tablist">
```
Comment (pgiraud) [inline thinkhazard/templates/report.jinja2]: `hazard-types-list` and `hasards` should probably be combined here.

**Comment by marionb** [comment]
`hazards` has no effect so I removed it.

**Comment by pgiraud** [comment]
Looks good. Please merge.

**Pull request merged**